doc/kernel/other/ring_buffers.rst

```diff
@@ -170,6 +256,47 @@ A data item is removed from a ring buffer by calling
         ...
     }
 
+Bytes data is copied out from a ring buffer by calling
```

Comment:
Change to:

    Data bytes are copied out from a **byte mode** ring buffer by calling